features/support/env.rb

```diff
@@ -121,12 +121,6 @@ def skip_below(os, version)
       FileUtils.mv '/tmp/kscrash.log', path
     end
   when 'ios'
-    # get_log can be slow (1 or 2 seconds) on device farms
```

Comment:
Removed as Maze Runner now also does this and there is the suspicion that they interfere with each other.